pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -33,15 +34,22 @@ def __call__(self, time: float):
         infection event occurs on individual and cell infectiousness
         and susceptibility.
 
-        :param time: Current simulation time
-        :type time: int
+        Parameters
+        ----------
+        time : float
+            Current simulation time
+
```

Comment:
I was updating docs, and noticed you are handling time as an int, but it should be a float to be compatible with Luke' changes

Reply:
Okey cool, is that all sorted?

Reply:
The docs are, the code still generates/uses time as an int.